ic-helper.ps1

```diff
@@ -66,26 +66,46 @@ function Find-GitRepositories {
         [string]$command,
         [switch]$force
     )
+
+ # Check if the path exists
+
+$foundRepository = $false
+
+try {
+    if (-not (Test-Path $path)) {
+        throw "The specified path '$path' does not exist."
+    }
     $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+}
+
+catch {
```

Comment:
You must specify exact type of error after `catch` keyword

Reply:
I did it in the comments now, hope it is ok